documentation/asciidoc/topics/proc_configuring_sites_manually.adoc

```diff
@@ -0,0 +1,106 @@
+[id='configuring_sites-{context}']
+= Specifying {brandname} Cluster Hostnames for Cross-Site Replication
+Specify hostnames and ports for {brandname} clusters so they can establish connections and form cross-site views.
+
+.Prerequisites
+
+* Ensure that you have the correct hostnames and port numbers for each {brandname} cluster that you plan to configure as a backup location.
+
+.Procedure
+
+. Create an **Infinispan** CR for each {brandname} cluster.
+. Specify the name of the local site with `spec.service.sites.local.name`.
+. Provide the name, host, and port for each {brandname} cluster that acts as a backup location with `spec.service.sites.locations`.
++
+The following are example **Infinispan** CR definitions for **LON** and **NYC**:
++
+* **LON**
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: {example_crd_name}
+spec:
+  replicas: 3
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: LON
+        expose:
+          type: LoadBalancer
+      locations:
+        - name: LON
+          host: infinispan-lon.myhost.com
+          port: 7900
+        - name: NYC
+          host: infinispan-nyc.myhost.com
+          port: 7900
+----
++
+* **NYC**
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: {example_crd_name}
+spec:
+  replicas: 2
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: NYC
+        expose:
+          type: LoadBalancer
+      locations:
+        - name: NYC
+          host: infinispan-nyc.myhost.com
+          port: 7900
+        - name: LON
+          host: infinispan-lon.myhost.com
+          port: 7900
+----
++
+. Adjust logging levels for cross-site replication as follows:
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  logging:
+    categories:
+      org.jgroups.protocols.TCP: error
+      org.jgroups.protocols.relay.RELAY2: fatal
+----
++
+The preceding configuration decreases logging for JGroups TCP and RELAY2
+protocols to reduce excessive messages about cluster backup operations, which
+can result in a large number of log files that use container storage.
++
+. Configure nodes with any other {datagridservice} resources.
+. Apply the **Infinispan** CRs.
+. Check node logs to verify that {brandname} clusters form a cross-site view, for example:
++
+[source,options="nowrap",subs=attributes+]
+----
+$ {oc_logs} {example_crd_name}-0 | grep x-site
```

Comment:
could be in a separate file since the messages are the same for "automatic" and "manual" cross-site.